xwiki-platform-core/xwiki-platform-scheduler/xwiki-platform-scheduler-ui/src/main/resources/XWiki/SchedulerJobSheet.xml

```diff
@@ -39,48 +39,58 @@
   <minorEdit>false</minorEdit>
   <syntaxId>xwiki/2.1</syntaxId>
   <hidden>true</hidden>
-  <content>{{velocity}}
-#if($doc.getObject('XWiki.SchedulerJobClass'))
-#set($job = $doc.getObject("XWiki.SchedulerJobClass"))
-**$services.localization.render('xe.scheduler.job.name')**
+  <content>{{velocity output='false'}}
+#macro (displayJobProperty $obj $propName)
+  #set ($class = $obj.xWikiClass)
+  #set ($prettyName = $class.get($propName).translatedPrettyName)
+  &lt;dt&gt;
```

Comment:
It does seem that you have manually added `&lt;` and `&gt;` in order to escape `<` and `>`. This also means that you have worked on the content of the page with a text editor or directly pasted into the XML file of the page, which is something you must avoid in the future. The problem is that, while you have manually fixed the `<` character, there can be other cases inside the document's content that need escaping and it is not good for you to do them manually. Instead, you should always work on the wiki page, in the browser, save and export the page before committing it to git. This way, any needed escapes will be performed by XWiki and you will rest assured that you're not missing anything.

A good example here is the fact that the `>` character does not need escaping (`&gt;`). As a result, once someone will export this page and try to commit, they will see that XWiki did not escape the `>` character and their commit will generate changes beyond the expected ones.

Thus, to be sure, you need to do a final export of this page from your wiki and commit the changes it will produce (i.e. at least the unescaping of `&gt;` back to `>`).

Reply:
Would be great to have the `mvn xar:format` doing the same export as XWiki.

Reply:
Actually, I don't see how it could do that, since XWiki is the only one that has the content in its original version and could be able of doing the proper exporting. `xar:format` gets as input an xml document on which is hard to tell if a character should be escaped or not, depending on the author's intention and if XWiki already escaped it or not.

It would seem that `xar:format` is already doing some escaping, but in an old fashion, not synchronized with what XWiki is doing. So XWiki is not escaping `>` but `xar:format` is... which is wrong.

Reply:
I actually pasted some rows into the XML file of the page(my mistake), but that happened 4 commits ago. Since then I used only the file exported from the wiki.